plugin/output/s3/mock/s3.go

```diff
@@ -44,7 +44,7 @@ func (m *MockObjectStoreClient) BucketExists(bucketName string) (bool, error) {
 }
 
 // BucketExists indicates an expected call of BucketExists.
-func (mr *MockObjectStoreClientMockRecorder) BucketExists(bucketName interface{}) *gomock.Call {
```

Comment:
Is this the output of MockGen?

// Code generated by MockGen. DO NOT EDIT.

Reply:
My bad. I regenerate mock